index.js

```diff
@@ -96,8 +105,26 @@ const ScrollableTabView = React.createClass({
     }
   },
 
+  updateSceneKeys(children, sceneKeys = [], currentPage) {
```

Comment:
yes, distinct keys, which are added only on first time click.
Lets say out of 5 given tabs, user goes to tab2 for the first time from initial tab tab 1, and then tab3 for the first time ,now you have 3 keys .User doesnt open tab4/5 but goes back to tab2 again, then new key shouldnt be added, only 3 keys are there. Only when user scrolls to tab4/5 for the first time then it will be added. Once 5 keys are there, its not updated anymore